src/communication_interface.cc

```diff
@@ -111,6 +118,15 @@ bool CommunicationInterface::HasNewPlan() {
   return robot_plan_.has_plan_data_;  // is atomic
 }
 
+bool CommunicationInterface::IsSimulatingControlException() {
```

Comment:
Since the function is named `IsSimulatingControlException`, it might be a good idea to not reset it here?